draft-birkholz-cose-receipts-ccf-profile.md

```diff
@@ -67,7 +67,7 @@ normative:
 
 --- abstract
 
-This document defines a new verifiable data structure type for COSE Signed Merkle Tree Proofs specifically designed for transaction ledgers produced by Trusted Execution Environments (TEEs), such as the Confidential Consortium Framework ({{CCF}}) to provide stronger tamper-evidence guarantees.
+This document defines a new verifiable data structure type for COSE Signed Merkle Tree Proofs specifically designed for transaction ledgers produced via Trusted Execution Environments (TEEs), such as the Confidential Consortium Framework ({{CCF}}) to provide stronger tamper-evidence guarantees.
```

Comment:
@JAG-UK is this is a sufficient clarification? Should we expand and explain that it's software that's running in the TEE that produces the ledger, or do we think that's reasonably obvious?